tests/integration/backup_tests/test_backups.py

```diff
@@ -105,7 +111,8 @@ async def test_blocked_incorrect_conf(ops_test: OpsTest) -> None:
 @pytest.mark.abort_on_fail
 async def test_ready_correct_conf(ops_test: OpsTest) -> None:
     """Verifies charm goes into active status when s3 config and creds options are correct."""
-    db_app_name = await helpers.app_name(ops_test)
+    db_app_name = await get_app_name(ops_test)
+    await helpers.set_credentials(ops_test, cloud="AWS")
```

Comment:
Should `AWS` be parametrized here?

Reply:
parametrised in what sense?

Reply:
With regards to the cloud substrate (like you do in other tests). I guess my question is: is this test supposed to only run on AWS or other clouds

Reply:
This one does not really care. We can improve the test to add "GCP", but I think it is better to do that a a part of separate PR (specifically for tests updates)

Reply:
This one does not really care. We can improve the test to add "GCP", but I think it is better to do that a a part of separate PR (specifically for tests updates)

Reply:
Sure, if SQA doesn't need to test on GCP - we can push it to another PR

Reply:
@Mehdi-Bendriss testing with GCP + AWS is done in the other tests. When @delgod and I first designed this test suite we intended for the first tests to perform basic operations on AWS, and deemed it was not necessary to do both

However, it seems to me that this line of code is not necessary, as in. the previous test we set AWS credentials, so I will remove this line